pallets/author-noting/src/lib.rs

```diff
@@ -437,3 +428,91 @@ impl<T: Config> LatestAuthorInfoFetcher<T::AccountId> for Pallet<T> {
         LatestAuthor::<T>::get(para_id)
     }
 }
+
+pub struct RelayStuff;
```

Comment:
Very nice so far! I like it. A few nitpicks:

`ParaStuff` -> ParaStateReader

`RelayStuff` -> RelayStateReader

decouple `self_para_id` into a different trait, or for now even ignore it if you want

Move to primitives